.travis.yml

```diff
@@ -5,6 +5,10 @@ sudo: true
 before_install:
   - scripts/install_toxiproxy.sh
 
+script:
+  - bundle exec rake
+  - bundle exec rubocop
```

Comment:
I can try. But rubocop Rake task has this annoying property of always running before the test. I don't want the tests not to be run if syntax is not ok.

Anyway let's try